cwf/gateway/integ_tests/gx_reauth_test.go

```diff
@@ -325,8 +325,8 @@ func TestGxReAuthWithMidSessionPolicyInstallAndRemoval(t *testing.T) {
 
 	// Generate over 80% of the quota to trigger a CCR Update
 	req := &cwfprotos.GenTrafficRequest{
-		Imsi:   imsi,
-		Volume: &wrappers.StringValue{Value: *swag.String("450K")},
+		Imsi:    imsi,
+		Volume:  &wrappers.StringValue{Value: *swag.String("450K")},
```

Comment:
*swag.String("450K") is just "450" I think

Reply:
we can revise all those values in future PRs. We have many instances with swag string

Reply:
ok, i modified them in this PR ;)